lib/_layouts/_base.njk

```diff
@@ -3,88 +3,123 @@
 {% from "hero/macro.njk" import hero as appHero %}
 {% from "appFooter/macro.njk" import appFooter as oglFooter %}
 {% from 'back-link/macro.njk' import backLink %}
+{% from 'cookie-banner/macro.njk' import cookieBanner %}
 {% set htmlClasses = 'no-js' %}
 {% set homeUrl = '/' | url %}
 {% set containerClasses = containerClasses | default("app-width-container") %}
 <html lang="{{ htmlLang | default('en') }}">
 
-{% block head %}
-<head>
-  {% block headStart %}
+  {% block head %}
+    <head>
+      {% block headStart %}{% endblock %}
+
+      {% block headMeta %}
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta name="description" content="{{ description | markdown("inline") | striptags(true) }}">
+        <meta name="robots" content="{{ robots | default('noindex') }}">
+      {% endblock %}
+
+      {% block pageTitle %}
```

Comment:
I'll revert the formatting changes back because you're right it doesn't have anything to do with GA, but the standard for formatting is to use soft tabs with 2 spaces [coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/main/docs/contributing/coding-standards.md#html)

Reply:
Think the playbook needs all its Nunjucks / html format standardised with stricter lint rules. One for discussions.